app/Composers/AppComposer.php

```diff
@@ -50,5 +51,7 @@ public function compose(View $view)
         } else {
             $view->withSiteTitle(Config::get('setting.app_name'));
         }
+
+        $view->withFontSubset(Config::get('langs.'.App::getLocale().'.subset', 'latin'));
```

Comment:
Since the class is already using the `Config` facade, you could just call `Config::get('app.locale')`.